rekognition-apis/4-custom-celebrity-recognition.ipynb

```diff
@@ -0,0 +1,1156 @@
+    "        \n",
+    "createTableResponse = dynamodb.create_table(\n",
+    "    TableName=ddbTableName,\n",
+    "    KeySchema=[\n",
+    "        {\n",
+    "            'AttributeName': 'id',\n",
+    "            'KeyType': 'HASH'  #Partition key\n",
+    "        }\n",
+    "    ],\n",
+    "    AttributeDefinitions=[\n",
+    "        {\n",
+    "            'AttributeName': 'id',\n",
+    "            'AttributeType': 'S'\n",
+    "        },\n",
+    "\n",
+    "    ],\n",
+    "    ProvisionedThroughput={\n",
+    "        'ReadCapacityUnits': 10,\n",
+    "        'WriteCapacityUnits': 10\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "display(createTableResponse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### List DynamoDB Tables in your account to see newly created table "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# List DynamoDB Tables\n",
+    "# Let us look at list of our DynamoDB tables again to make sure that table we just created appears in the list.\n",
+    "\n",
+    "listTablesResponse = dynamodb.list_tables()\n",
+    "display(listTablesResponse[\"TableNames\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Rekogniton Collection to store faces\n",
+    "***\n",
+    "In this step we will create a Rekognition Collection.\n",
+    "\n",
+    "Amazon Rekognition can store information about detected faces in server-side containers known as [collections](https://docs.aws.amazon.com/rekognition/latest/dg/collections.html). You can use the facial information that's stored in a collection to search for known faces in images, stored videos, and streaming videos. In this section you will learn how you can create and manage Rekognition Collections."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### List Rekognition Collections"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# List Rekognition Collections\n",
+    "# Let us first see if we have already created any Rekognition collections in our account.\n",
+    "# If there is not an existing Rekognition in your account, you will see empty list\n",
+    "# otherwise you will a list with names of rekognition collections and face model version.\n",
+    "\n",
+    "listCollectionsResponse = rekognition.list_collections()\n",
+    "\n",
+    "display(listCollectionsResponse[\"CollectionIds\"])\n",
+    "display(listCollectionsResponse[\"FaceModelVersions\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create new Rekognition collection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#cids = listCollectionsResponse[\"CollectionIds\"]\n",
```

Comment:
This entire block is commented out. It might be more clear to either remove it, add a paragraph/comment to explain when a user might wish to run it